src/components/Layout/Layout.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Outlet } from 'react-router-dom';
+
+import styles from './Layout.module.css';
+
+// eslint-disable-next-line import/no-unresolved
+import Navbar from '../Navbar/Navbar';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../Navbar/Navbar"